tests/packages/browser.smash

```diff
@@ -915,8 +915,9 @@ Open test page [
             }
 
                 Switch to iframe [#frame]
-                    Verify ['Example Domain'] is visible
-                        Verify ['foobar'] is not visible
+                    Wait '2' secs
```

Comment:
Is the `await` In `Switch to iframe` not working for you?

Reply:
Which browsers are you seeing this in?

Reply:
I could only repro this in Safari actually. Also tried Chrome and FF on Mac, and no issue.

Reply:
I added a 100 ms wait in `Switch to iframe` in version 1.5.6. Please let me know if that's enough of a wait. It fixed the issue here.